Gemfile.lock

```diff
@@ -143,7 +138,6 @@ DEPENDENCIES
   rouge
   sassc
   therubyracer
-  video_info
```

Comment:
video info was a plugin that made it easier to get the still image for the video. api changed and no longer needed  a long hash that was different from video id. so video_info no longer needed. removed it.